pages/Getting Started/Quick-start.md

```diff
@@ -55,26 +55,26 @@ usually not ran through the user account.
 {{< hint type=important >}}
 It is highly recommended to copy the desktop file to e.g. `wrapped_hl.desktop` instead of editing
 the provided one, as many package managers (and `sudo make install`) will **overwrite**
-the desktop file on updates.
+the desktop file on updates. If you manually build Hyprland, consider using `sudo make cleaninstall` to preserve `hyprland.deskop`
 {{< /hint >}}
 
-## Immediate
+# Immediate
 
 **OMG MY SCREEN IS BROKEN, FLASHY TEARY!** -> see the bottom of this page
 
 Once you log in, you'll be greeted with a yellow warning that will give you some
 basic keybind info of your pregenerated config.
 
-I recommend you use the config provided in `examples/hyprland.conf` though.
+It's better to use the config provided in `examples/hyprland.conf` though.
```

Comment:
actually, we can remove this line. Both configs have been the exact same for quite some time now.